### DISCUSSION TITLE
Are there an option to keep the screen unlocked while I'm connected via remote?

### DISCUSSION THREAD

**Discussion by ethpsantos** [rustdesk/rustdesk#3732] [Q&A] [answered]
Hello.
I'm coming from AnyDesk that become unsable with the last updates and I tried to use Rustdesk last year but found out the Key Map options were really weird and most my accents and differente key didn't work (ABNT PT-BR).
I was really happy when I found out the key map now works fine in Nightly version.

But I have one single issue yet..
I use to control another work notebook from my own desk but it is not in my front... so from my main notebook I can control it... I can see the screen and what is happening both via the Rustdesk or the own notebook monitor.

The issue is when I do something in my own notebook and left the Rustdesk without do nothing it will lock the screen.
It should be simple to disable the lock in the Windows of that notebbok but I'm not adminstrator (I run rustdesk via portable).

AnyDesk have an option to keep screen unlocked and it worked while I has a remote session it won't ever lock (both notebooks are always in power so there is no issue with bateries).

Are there any option like that in Rustdesk to keep the screen unlocked while remote session is opened? I tried to find but couldn't find anything.

PS. You guys did an amazing job from 1.19 to 1.20... it is a complete redesign of the App... way better and clean than the old design.

Thanks in advance.

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/issues/230#issuecomment-1422615080
